api/fixtures/example.go

```diff
@@ -408,6 +409,13 @@ func (o ExampleOptions) Resources() *ExampleResources {
 		},
 	}
 
+	if o.BaseDomainPrefix == "none" {
+		// set empty prefix explicitly
+		cluster.Spec.DNS.BaseDomainPrefix = pointer.String("")
+	} else if o.BaseDomainPrefix != "" {
+		cluster.Spec.DNS.BaseDomainPrefix = pointer.String(o.BaseDomainPrefix)
+	}
```

Comment:
So it looks like we need to wrap all this in a nil check, since the new field is a pointer type, and the field won't necessarily be present yet on existing deployments.